### DISCUSSION TITLE
EpiNow2 and data format

### DISCUSSION THREAD

**Discussion by HAKGH** [epiforecasts/EpiNow2#294] [General] [answered]
Hello,
Can I ask whether EpiNow2 can work on data sets that have weekly or monthly counts Nostradamus of daily counts to minimize calculation time? And how can it be done?
Best regards,
Hanan

**Comment by seabbs** [comment, marked as answer]
It can work with these counts but internally will run on a daily time-scale and so this won't reduce runtimes.

**Reply by HAKGH** [reply]
How? what do we have to modify in the code?
Thanks

**Reply by seabbs** [reply]
You modify `week_length` to that of your aggregation. Note that this will still estimate a day of the week effect so for long timespans (i.e months) it is likely not very efficient. I am not sure anything useful can really be got from monthly counts though so perhaps this is not an issue. For weekly counts you should essentially be able to use as is though you may need to modifiy the limit for cases corrected when zero (i.e `zero_threshold` in `estimate_infections`).

**Reply by seabbs** [reply]
If your interest is primarily about reducing computation times though a better bet might be to use a different assumption for the gaussian process (see the options for `rt_opts`) or to use the backcalc/deconvolution method or to fit to a shorter time-series if your interest is in real-time estimates.

**Comment by HAKGH** [comment]
Thank you
